js/nodes/processes/load_process_list_pressure.js

```diff
@@ -0,0 +1,54 @@
+
+function AssignScalarVariableToConditionsProcess() {
+    this.addInput("model_part_name","string");
+    this.addInput("interval","process_array");
+
+    this.properties = {
+        "python_module" : "assign_scalar_variable_to_conditions_process",
+        "kratos_module" : "KratosMultiphysics",
+        "Parameters"    : {
+            "model_part_name" : "Structure.COMPUTE_HROM.SurfacePressure3D_Pressure_on_surfaces_Auto4",
+            "variable_name"   : "POSITIVE_FACE_PRESSURE",
+            "value"           : "(-15000000*13) + t*15000000",
+            "interval"        : [0.0,"End"]
+        }
+        
+    };
+
+        var that = this;
+        this.variable_name = this.addWidget("text","VariableName", "", function(v){}, {} );
+        this.value = this.addWidget("text","Value", "", function(v){}, {});
+
+
+        this.addOutput("Process","process");
+
+        this.size = this.computeSize();
+        this.serialize_widgets = true;
+
+    }
+
+    AssignScalarVariableToConditionsProcess.title = "Load process list pressure";
+    AssignScalarVariableToConditionsProcess.desc = "Node to specify a load process";
+
+    AssignScalarVariableToConditionsProcess.prototype.onExecute = function() {
+        myoutput = this.properties
+    // model_part_name
+    if (this.getInputData(0) != undefined) {
+        myoutput["Parameters"]["model_part_name"] = this.getInputData(0)
+    } else {
+        myoutput["Parameters"]["model_part_name"] = this.properties["Parameters"]["model_part_name"]
+    }
+    // interval
+    if (this.getInputData(1) != undefined) {
+        myoutput["Parameters"]["interval"] = this.getInputData(1)
+    } else {
+        myoutput["Parameters"]["interval"] = this.properties["Parameters"]["interval"]
+    }
+  
+        myoutput["Parameters"]["variable_name"] = this.variable_name.value
+        myoutput["Parameters"]["value"] = this.value.value
+
+        this.setOutputData(0,  myoutput);
+    };
+
+    LiteGraph.registerNodeType("processes/LoadProcessListPressure", AssignScalarVariableToConditionsProcess);
```

Comment:
Ok, but for generic process in the Kratos Core, its probably better if you generate them through the https://github.com/KratosMultiphysics/Flowgraph/blob/master/js/nodes/processes/process_parser.js.

That will create the process node defining only the property parameters, if they don't expect any special input (tables, vectors, etc...)

Reply:
If we keep it as a separate code, the name should be the name of the process. In this case `AssignScalarVariableToConditionsProcess.js` or something like that, as it can be used to initialize other variables than the pressure.

Once you have it in the example file, it will look to something like (taken from the example in the last commit):
```json
{"id":215,"type":"processes/AssignScalarVariableToConditionsProcess","pos":[253,-23],"size":{"0":210,"1":102},"flags":{},"order":13,"mode":0,"inputs":[{"name":"model_part_name","type":"string","link":333},{"name":"interval","type":"process_array","link":335}],"outputs":[{"name":"Process","type":"process","links":[337]}],"properties":{"python_module":"assign_scalar_variable_to_conditions_process","kratos_module":"KratosMultiphysics","Parameters":{"model_part_name":"Structure.COMPUTE_HROM.SurfacePressure3D_Pressure_on_surfaces_Auto2","variable_name":"POSITIVE_FACE_PRESSURE","value":"0","interval":[0,"End"]}},"widgets_values":["POSITIVE_FACE_PRESSURE","0"]},
```